app/lib/actions/pulp/repository/presenters/docker_presenter.rb

```diff
@@ -31,7 +31,7 @@ def humanized_details
 
             if content_started?(download_details)
               if items_total(download_details) > 0
-                ret << (_("New images: %{count}.") % {:count => count_summary})
+                ret << (_("New blobs: %{count}.") % {:count => count_summary})
```

Comment:
@jlsherrill, It looks like the current reporting doesn't allow us to do that.  I'll open a ticket with pulp to request better reporting; however, before doing so, I want to sync each of the supported repository types to see how they compare.